OpenSim/Simulation/Model/Model.h

```diff
@@ -682,6 +683,11 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Model, ModelComponent);
      * This call invalidates the dynamics of the model and invalidates the
      * value of the controls until they are marked as valid when the update
      * is completed (@see markControlsAsValid)
+     *
```

Comment:
While you're here could you update the doxygen for this `@see markControlsAsValid`? It looks incorrectly formatted in doxygen. The `@see` creates an indented block, and the parentheses are moved around.